src/ui/src/js/controllers/command_view.js

```diff
@@ -60,10 +63,19 @@ export default function commandViewController(
       namespace: requestPrototype['namespace'] || $scope.system.namespace,
       system: requestPrototype['system'] || $scope.system.name,
       system_version: requestPrototype['system_version'] || $scope.system.version,
-      instance_name: requestPrototype['instance_name'] || $scope.model['instance_name'] ||
-          'default',
+      instance_name: requestPrototype['instance_name'] || $scope.model['instance_name']
     };
 
+    // If a system has more than one instance this will be undefined on initial page
+    // load. We could just let this continue as normal, but the backend would return
+    // an error like "Could not find instance with name 'None' in system". That's not
+    // the best, so this special handling makes the error message a little nicer.
+    if (!request.instance_name) {
+      let deferred = $q.defer();
+      deferred.reject("Please select an instance");
```

Comment:
Instead of this error, I'm getting an error in the small circle tooltip that says `Couldn't populate choices from Undefined`. Selecting a running instance does not remove this error, nor correctly populate choices with a new request.

Reply:
I'm guessing this related to the the changes in the form-addons repo seen [here](https://github.com/beer-garden/angular-schema-form-addons/pull/37/files#diff-bc6b134f359dc185bece7f778ad5d71d88b95764684f1c5d2522d42a26468207R329).  I tested this PR against the form-addons from that one and it seems to work as expected.